client/dashboard/components/code-highlighter/style.scss

```diff
@@ -0,0 +1,17 @@
+.github-workflow-code-highlighter {
+	font-size: $font-body-small;
+	height: fit-content;
+	max-height: 30lh;
+	border-radius: 4px;
+	background: transparent !important;
+	position: relative;
+
+	.language-yaml {
+		color: var( --studio-wordpress-blue-60 ) !important;
```

Comment:
In the new Hosting Dashboard, we don't want to use these color studio variables I think 🤔

Reply:
I added the CSS colors of Studio for the syntax highlight for now. Could you let me know if there are any rules around the colors that can be used?

I think these colors are still matching the overall colors of the page better than the original prism stylesheet, but we could also potentially remove all style overrides.

| With style overrides | Without style overrides |
|--------|--------|
| <img alt="CleanShot 2025-09-22 at 16 51 11@2x" src="https://github.com/user-attachments/assets/7e34c343-9b8b-478b-9dfe-8a3c958e2316" /> | <img alt="image" src="https://github.com/user-attachments/assets/048dbd07-4710-4e4f-8ddb-8b2e042c0a5c" /> |